pages.th/osx/pbpaste.md

```diff
@@ -0,0 +1,12 @@
+# pbpaste
+
+> ส่งเนื้อหาของคลิปบอร์ดไปยังผลผลิตมาตรฐาน (`stdout`) ซึ่งเทียบได้กับการกดปุ่ม Cmd + V บนแป้นพิมพ์.
```

Comment:
`ซึ่งเทียบได้กับการกดปุ่ม Cmd + V บนแป้นพิมพ์.`  ( which is equivalent to hitting Cmd + V on your keyboard.)

Similarly, Can you add this to [English page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/pbpaste.md) too?

Reply:
Added it to the original English page